manifests/manifest-firefox.json

```diff
@@ -47,6 +47,7 @@
     "optional_permissions": [
         "<all_urls>",
         "clipboardWrite",
+        "contextMenus",
```

Comment:
This is not an allowed optional permission in Firefox. I think it may be best to just remove Firefox support for this feature rather than adding a required permission.

Reply:
Updated